app/src/main/java/com/github/houseorganizer/houseorganizer/shop/ShopList.java

```diff
@@ -37,23 +31,24 @@ public void removeItem(int index){
         items.remove(index);
     }
 
-    public void removedPickedUpItems(){
+    public void removePickedUpItems(){
         items.removeIf(ShopItem::isPickedUp);
     }
 
     public boolean isEmpty(){
         return items.isEmpty();
     }
 
-    //Getters
-    public String getListName() {
-        return name;
+    public List<ShopItem> getItems(){
+        return new ArrayList<>(items);
     }
 
-    public User getOwner() {
-        return owner;
+    public void setItems(List<ShopItem> items) {
```

Comment:
This method sets the entire list of items, what if I only want to change one of them. Do we have a case for that?

Reply:
Yes if you want to change just one of them you can do
 `shopList.getItem(index).setName("")` for example and then
 `shopList.updateItems` to upload the changes to Firebase ^^